Makefile

```diff
@@ -69,10 +80,21 @@ fix: ## Fix linting issues in the code
 	${DOCKER_COMPOSE} run --rm app npm run lint:fix
 
 test: export TARGET = dev
-test: ## Run the tests
-	${DOCKER_COMPOSE} run --rm app npm run test
+test: ## Run all the tests
+	$(MAKE) start-db
+	${DOCKER_COMPOSE} run --rm app npm run test; EXIT=$$?; ${DOCKER_COMPOSE} down; exit $$EXIT
```

Comment:
I forgot to change it after we introduced it, but `EXIT` should be lowercase as uppercase in shell is for environment variables rather than local variables?